solutions/swb-app/src/projects/disassociateProjectUserRequest.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { z } from 'zod';
+
+// eslint-disable-next-line @rushstack/typedef-var
+export const DisassociateUserToProjectRequestParser = z
```

Comment:
nit: does it make more sense to say DisassociateUserFromProjectRequestParser

Reply:
fixed